nemo/core/classes/modelPT.py

```diff
@@ -177,12 +178,39 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
                     f"Test config : \n{OmegaConf.to_yaml(self._cfg.test_ds)}"
                 )
 
+        # Create list of lists for val and test outputs to support multiple dataloaders
+        self.validation_step_outputs = (
+            []
+        )  # Initialize an empty list as sometimes self._validation_dl can be None at this stage
```

Comment:
can we move this comment above line 182 so we don't have the extra formatting for a long line?

Reply:
Fixed.